scripts/update-authors.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+// original comes from https://github.com/nodejs/node/blob/master/tools/update-authors.js
```

Comment:
Meanwhile the original Node script seems to have been updated

Reply:
It doesn't. 
I've removed mailmap part from the original script.

Reply:
3d36c93d3f576eaccb9c9c1a2e3434d76527681f

Reply:
> It doesn't. 
> I've removed mailmap part from the original script.

Reply:
> It doesn't. 
> I've removed mailmap part from the original script.

Reply:
> It doesn't. 
> I've removed mailmap part from the original script.

Reply:
> Meanwhile the original Node script seems to have been updated

Reply:
> It doesn't. 
> I've removed mailmap part from the original script.